src/ai/ai.cpp

```diff
@@ -70,6 +70,13 @@ void update_ai_general_status(sys::state& state) {
 		bool threatened = defensive_str < safety_factor * greatest_neighbor;
 		state.world.nation_set_ai_is_threatened(n, threatened);
 
+		// account for domestic rebel threats
+		auto militancy = state.world.nation_get_demographics(n, demographics::militancy);
+		auto total_pop = state.world.nation_get_demographics(n, demographics::total);
+		if(total_pop / militancy >= state.defines.mil_to_join_rebel * 0.75f) {
+			state.world.nation_set_ai_is_threatened(n, true);
```

Comment:
Being threatened is just for making alliances. And allies won't fight your rebels. So this is counterproductive